lib/probes_.h

```diff
@@ -98,9 +104,20 @@ static inline void h2o_probe_log_request(h2o_req_t *req, uint64_t req_index)
     }
 }
 
-static inline void h2o_probe_log_response_status(h2o_req_t *req, uint64_t req_index)
+static inline void h2o_probe_log_response(h2o_req_t *req, uint64_t req_index)
 {
     H2O_PROBE_CONN(SEND_RESPONSE_STATUS, req->conn, req_index, req->res.status);
+    if (H2O_CONN_IS_PROBED(SEND_RESPONSE_HEADER, req->conn)) {
+        h2o_iovec_t cl_buf;
+        cl_buf.base = h2o_mem_alloc_pool(&req->pool, char, req->res.content_length);
+        cl_buf.len = sprintf(cl_buf.base, "%zu", req->res.content_length);
+        h2o_probe_response_header(req, req_index, H2O_TOKEN_CONTENT_LENGTH->buf, cl_buf);
```

Comment:
We should be logging the content-length header only when `req->res.content_length` holds a valid value. SIZE_MAX is the value you need to consult, please refer to the the codes in lib/http1.c, lib/http2/hpack.c, etc.

Reply:
Thank you for the examples! addressed in: 4eb9ada7e